### DISCUSSION TITLE
Should kulala-fmt (try to) format the body of the request?

### DISCUSSION THREAD

**Discussion by gorillamoe** [mistweaverco/kulala-fmt#5] [Polls]
As discussed here: #3, it might be a good addition, but I'm afraid it might break some things we can't really foresee.

JSON parsing is easy and making sure its indentation is always the same, but I'm afraid of the multipart form-data stuff somehow.

What do you guys and girls think?

**Comment by franroa** [comment]
Is possible to add some kind of "injected language" to http for the Jason part and use a Json formatter?

**Comment by gorillamoe** [comment]
Formatting of the body is already implemented.
